src/Http/Routing/test/testassets/Benchmarks/Benchmarks.csproj

```
@@ -1,9 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
-    <TargetFramework Condition="'$(BenchmarksTargetFramework)' != ''">$(BenchmarksTargetFramework)</TargetFramework>
-    <UseP2PReferences Condition="'$(UseP2PReferences)'=='' AND '$(BenchmarksTargetFramework)'==''">true</UseP2PReferences>
+    <UseP2PReferences Condition=" '$(UseP2PReferences)'=='' ">true</UseP2PReferences>
```

I think you can add the AND, this way it will only use P2P if it's set, and not running in benchmarks

```suggestion
    <UseP2PReferences Condition="'$(UseP2PReferences)'=='' AND '$(BenchmarksTargetFramework)'==''">true</UseP2PReferences>
```